feat(forms): add `Value.Selection` component (#3857)
dnbexperience/eufemia: packages/dnb-design-system-portal/src/docs/uilib/extensions/forms/Value/Selection.mdx

---
title: 'Selection'
description: '`Value.Selection` is a component for displaying a string value based on a user selection.'

> componentType: 'base-value'

Reviewer: What does this?
Never seen this prop before

Reviewer: Never mind :)

> description: '`Value.Selection` is a wrapper component for displaying string values, with user experience tailored for selection values.'

Reviewer 2: ```suggestion
description: '`Value.Selection` is a component for displaying a string value based on a user selection.'
```
